srcpkgs/gptfdisk/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'gptfdisk'
 pkgname=gptfdisk
-version=1.0.6
+version=1.0.8
 revision=1
 build_style=gnu-makefile
 makedepends="ncurses-devel popt-devel icu-devel libuuid-devel"
```

Comment:
```suggestion
makedepends="ncurses-devel popt-devel libuuid-devel"
```

Reply:
The icu dependency can be safely removed.  It looks like it is optional and not enabled by default in the Makefile anyways.

Reply:
Done.